applications/optimization/integer_linear_programming/integer_linear_programming.ipynb

```diff
@@ -45,28 +45,6 @@
     "We go through the steps of solving the problem with the Classiq platform, using QAOA algorithm [[2](#QAOA)]. The solution is based on defining a pyomo model for the optimization problem we would like to solve."
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    optimization_result.sort_values(by="cost").head(5)
</pre><p>:) This the same lie as before - sorting the full set of results by their cost (where the optimal solution pretty far down in terms of the number of samples..)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>